content/en/blog/2022-03-21-tools-to-manage-helm-declaratively.md

```diff
@@ -0,0 +1,86 @@
+
+Kubernetes provides a means to do both [declarative and imperative management of resources](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/). As the Kubernetes community tends to prefer declarative management, when possible, the rest of this post is going to focus on declarative tools you can use with Helm.
+
+## Tools
+
+The Kubernetes ecosystem has produced numerous projects of various styles to help you declaratively manage your Helm releases. To illustrate the options we will look at sister projects to Helm in the Cloud Native Computing Foundation (CNCF) and some more general open source projects. You can find more options in the [CNCF Landscape](https://landscape.cncf.io/).
+
+### CNCF Projects
+
+The scope of this section is limited to [graduated and incubating](https://www.cncf.io/projects/) CNCF projects. There are over 100 CNCF projects and many of them are [sandbox projects](https://www.cncf.io/sandbox-projects/). You can learn more about the differences between types of projects in the [maturity level explanation](https://www.cncf.io/projects/#maturity-levels). The following projects are worth looking at:
+
+* [Flux Helm Controller](https://fluxcd.io/docs/components/helm/) - [Flux](https://fluxcd.io/) is a collection of projects that enable GitOps. One of the components provides a GitOps method to manage Helm releases.
+* [Argo CD](https://github.com/argoproj/argo-cd) - The [Argo](https://argoproj.github.io/) project defines itself as providing "Open source tools for Kubernetes to run workflows, manage clusters, and do GitOps right." Argo CD is focused on declarative continuous delivery and has the ability to work with Helm charts.
+
+### Other Projects
+
+There are many projects beyond the CNCF projects you can use to help you manage your Helm releases. The following set is an example and not exhaustive.
+
+* [Helmfile](https://github.com/roboll/helmfile) - A declarative spec for deploying Helm charts.
+* [Captain](https://github.com/alauda/captain) - A Helm controller.
+* [Terraform Helm provider](https://github.com/hashicorp/terraform-provider-helm) - Enables you to manage Helm charts through Terraform.
+* [Orkestra](https://azure.github.io/orkestra/) - Built on other tools in this list, Orkestra adds a robust dependency graph for a related group of Helm releases and their subcharts, as well as a reverse DAG for specifying dependency requirements for rollbacks.
+* [Fleet](https://github.com/rancher/fleet) - A GitOps tool chain that works with Kubernetes manifests, Helm charts, and Kustomize.
+
+### High-Level Tool Comparison
+
+There are some differences between the tools we've looked at so far. The following table provides some insight into their differences. This is not exhaustive and you should evaluate any tools you use yourself.
+
+| | Retains Helm release info | Supports Helm hooks | OCI support | Does not require Helm binary |
+| -- | -- | -- | -- | -- |
+| Flux Helm controller | ✅ | ✅ | ✅[^1] | ✅ |
```

Comment:
@yurrriq I'm reverting this one accidental change per https://github.com/scottrigby/helm-www/pull/1#pullrequestreview-943106136

```suggestion
| Flux Helm controller | ✅ | ✅ | 🚫[^1] | ✅ |
```